run_benchmarks.rb

```diff
@@ -32,21 +32,37 @@ def os
 end
 
 # Checked system - error if the command fails
-def check_call(command, verbose: false, env: {})
+def check_call(command, env: {})
   puts(command)
 
-  if verbose
-    status = system(env, command, out: $stdout, err: :out)
-  else
-    status = system(env, command)
-  end
+  status = system(env, command)
 
   unless status
     puts "Command #{command.inspect} failed in directory #{Dir.pwd}"
     raise RuntimeError.new
   end
 end
 
+# Call system, capture stderr, return that with result.
+def run_benchmark_command(command, env: {})
+  puts(command)
+
+  result = {}
+
+  IO.pipe do |reader, writer|
+    result[:success] = system(env, command, err: writer)
```

Comment:
Fair enough. It will occasionally be different when we hit unexpected cases -- that is, when we hit different exit codes. But reasonable to not bother with it.